cmd/piper.go

```diff
@@ -208,6 +210,8 @@ func addRootFlags(rootCmd *cobra.Command) {
 	rootCmd.PersistentFlags().StringSliceVar(&GeneralConfig.DefaultConfig, "defaultConfig", []string{".pipeline/defaults.yaml"}, "Default configurations, passed as path to yaml file")
 	rootCmd.PersistentFlags().BoolVar(&GeneralConfig.IgnoreCustomDefaults, "ignoreCustomDefaults", false, "Disables evaluation of the parameter 'customDefaults' in the pipeline configuration file")
 	rootCmd.PersistentFlags().StringVar(&GeneralConfig.ParametersJSON, "parametersJSON", os.Getenv("PIPER_parametersJSON"), "Parameters to be considered in JSON format")
+	rootCmd.PersistentFlags().StringVar(&GeneralConfig.HookConfig.ANSConfig.ServiceKey, "ansServiceKey", os.Getenv("PIPER_ansServiceKey"), "Service Key JSON needed for ANS")
```

Comment:
inconsistent:

please don't add hook configuration as parameter (as for Splunk and Sentry)
This will overload users.

Reply:
Removed.